backend/src/api/app.ts

```diff
@@ -708,6 +709,35 @@ authenticatedRoute.get(
   handlerToExpress(organizations.getAllV2)
 );
 
+
+//************* */
+//  Open Source Project Routes   //
+//************* */
+
+// get
+authenticatedRoute.get(
```

Comment:
Could you standardize the naming convention into the following?

Get: `/projects/:projectId`
Post: `/projects'
List: '/projects'
Delete '/projects/:projectId'

orgId can be passed as a parameter in the request body.